index.html

```diff
@@ -31,6 +31,138 @@ <h1 id="main-heading">Hey there. I'm Ulises <br><strong>I'm a Software Developer
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="work-card">
+      <div class="work-card-content">
+      <h3>Multi-Post Stories Gain+Glory</h3>
+      <ul>
+        <li class="card-work-item">Ruby on Rails</li>
+        <li class="card-work-item">CSS</li>
+        <li class="card-work-item">HTML</li>
+        <li class="card-work-item">Javascript</li>
+      </ul>
+      <button type="button" class="main-button">See Project</button>
+      </div>
+    </div>
+
   </section>
+
+  <article id="about-me-article">
+    <h2 class="headings-2">About Me</h2>
+    <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+    <button type="button" class="main-button">Get My Resume</button>
+
+    <div class="aboutme-card">
+      <img src="images/icon-languages.png" alt="circle language icon">
+      <h4>Languages</h4>
+      <ul>
+        <li class="card-list-item">Javascript</li>
+        <li class="card-list-item">Ruby</li>
+        <li class="card-list-item">HTML</li>
+        <li class="card-list-item">CSS</li>
+      </ul>
+    </div>
+
+    <div class="aboutme-card">
```

Comment:
- [ ] [REQUIRED] Please, use `<section>` instead of a `<div>` for semantic purposes ♻️. The idea here is that each main section 🔪 of the portfolio website is a section, the headline, work, and about. Please, only use `<div>` for styling purposes.